ansible/roles/rke/vars/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+kubectl_version: v1.15.2
+local_output_dir: "/tmp"
+rke_cli_version: v0.2.7
```

Comment:
Might be worth upgrading to **v0.3.0**. It supports the creation of k8s v1.15 clusters and the tainting of nodes.

Reply:
Nice - just release 12 days ago. Will update.